.gitignore

```diff
@@ -25,13 +25,7 @@ public/client
 public/system
 public/uploads
 
-plugins/*/
-plugins/Gemfile
-!plugins/*.rb
-!plugins/*.yml
-!plugins/*.md
-# NB: To reveal a plugin to hack on it locally, add a line here telling .gitignore to reveal it:
-# !plugins/my_awesome_plugin
+plugins/
```

Comment:
This seems way simpler than before.

Ive taken the "this is how you unignore notice out because I think people should use their own repo from from the start" I'd recommend people `git init` and commit their plugins rather than expose them to the loomio repo.